app/controllers/grades_controller.rb

```diff
@@ -202,6 +203,12 @@ def save_grade_and_comment_for_submission
     redirect_to controller: 'grades', action: 'view_team', id: params[:participant_id]
   end
 
+  def filter(string)
+    #TODO: remove testing hard coded value
+    sql_query = "select sub.* from (select t1.* from teams t1, teams_users tu, users u where t1.id = tu.team_id and tu.user_id = u.id and u.name like '6370' union select t2.* from teams t2 where t2.name like '6370') as sub" #+ :nameString
```

Comment:
Its a select statement so no potential for malicious sql commands running on the DB.